src/MetaMorpho.sol

```diff
@@ -616,9 +616,10 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
             uint256 supplyCap = config[id].cap;
             if (supplyCap == 0) continue;
 
-            uint256 supplyAssets = MORPHO.expectedSupplyAssets(_marketParams(id), address(this));
```

Comment:
shouldn't we rename this function in the morpho lib? I'm ready to bet that a lot of integrators will have the same issue

Reply:
What issue are you referring to ? Isn't this an issue to submit on the `morpho-blue` repo instead ?

Reply:
I see it really tied to the fact we need to under-estimate the maximum, so over-estimate every subtracted part to the maximum. Why would we create a specific function to over-estimate the supply assets of some account on Blue?

It seems very edge-casy